torchaudio/csrc/iir_cuda.cu

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#ifdef USE_CUDA
```

Comment:
Since it's kind of premise that `iir_cuda.cu` and `iir_cuda.h` are used only when USE_CUDA is enabled, so I prefer to not use `#ifdef USE_CUDA` in these files, and rather use it in `lfilter.cpp`, where the final product is composed.
What do you think?

Reply:
Agree.